astro_2.0/components/TransactionDetailsWidget/TransactionDetailsWidget.module.scss

```diff
@@ -43,15 +43,20 @@
   align-items: center;
   display: flex;
   flex: 1;
-  justify-content: flex-end;
+  justify-content: space-between;
   width: 100%;
 }
 
 .transactionDetails {
+  align-items: flex-end;
   display: flex;
   flex-basis: 42%;
-  justify-content: space-evenly;
+  justify-content: flex-start;
   width: 100%;
+
+  > div {
```

Comment:
We should avoid tightening of layout with styles. Usage of tags should be avoided if can be done by styles.